composer.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "spryker/app-webhook",
+  "type": "library",
+  "description": "AppWebhook module",
+  "license": "proprietary",
+  "require": {
+    "php": ">=8.1",
+    "spryker/glue-application-extension": "^1.0.0",
+    "spryker/kernel": "^3.30.0",
+    "spryker/log": "^3.0.0",
+    "spryker/message-broker": "^1.11.0",
+    "spryker/propel-encryption-behavior": "^0.1.1",
+    "spryker/secrets-manager": "^1.0.0",
+    "spryker/secrets-manager-extension": "^1.0.0",
```

Comment:
seems not used  in the package.please also  validate other dependencies (including req-dev)